src/include/xocl-axigate.h

```diff
@@ -20,7 +22,7 @@ enum xocl_axigate_ioctl_cmd {
 };
 
 /* the ep names are in the order of hardware layers */
```

Comment:
Question for @houlz0507 , do we still need to list the names in restricted order here? I thought we are now using the event and not looping over these names?

Reply:
Yes. With on 1RP flow, two AXI gates are probed in same partition. And in this case, we have to open only Lower level (PLP) gate and keep Upper level (ULP) gate closed.